Sleep Tracker/Sleep Tracker/View/SleepPageView/SleepSetupView.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+struct SleepSetupView: View {
+    @State var animate = false
+    @State private var currentDate = Date()
+
+    var body: some View {
+        VStack {
+            Image("Sleeping_astronaut_image")
```

Comment:
I'd like to have typed image names. Smth like:

```suggestion
            Image(.astronautImage)
```

Otherwise, it's easy to have a typo plus we follow the DRY principle.

Reply:
fixed